x/ibchooks/marker_hooks.go

```diff
@@ -96,6 +96,7 @@ func (h MarkerHooks) createNewIbcMarker(ctx sdktypes.Context, data transfertypes
 		allowForceTransfer,
 		[]string{},
 	)
+	h.getExistingSupply(ctx, marker)
```

Comment:
This should return the existing supply so it can be added here in this method to make it clear what is going on.  This current approach hides that the existing supply is being added to the amount that is inbound from the transfer.

Reply:
Yeah, I refactored it a minute ago.  I realized I mis-understood your first comment on my other refactor.